AzureBot/Dialogs/ActionDialog.cs

```diff
@@ -35,7 +36,7 @@ public class ActionDialog : LuisDialog<string>
         {
             string message = "Hello! You can use the Azure Bot to: \n";
             message += $"* List and select an Azure subscription (e.g. 'list all subscriptions', 'use the QA subscription')\n";

```

Comment:
Let's remove all of the (e.g ....) for all the commands since it is starting to get very long (and will continue to get longer over time).  We should hope that LUIS just detects things :)
Also - switch subscription works, so let's say "List, Switch, and Select and Azure subscription"